common/src/main/java/com/aliyun/emr/rss/common/network/server/BaseMessageHandler.java

```diff
@@ -17,24 +17,34 @@
 
 package com.aliyun.emr.rss.common.network.server;
 
-import java.nio.ByteBuffer;
-
-import com.aliyun.emr.rss.common.network.client.RpcResponseCallback;
 import com.aliyun.emr.rss.common.network.client.TransportClient;
+import com.aliyun.emr.rss.common.network.protocol.RequestMessage;
 
-/** An RpcHandler suitable for a client-only TransportContext, which cannot receive RPCs. */
-public class NoOpRpcHandler extends RpcHandler {
-  private final StreamManager streamManager;
+/**
+ * Handler for sendRPC() messages sent by {@link TransportClient}s.
+ */
+public class BaseMessageHandler {
```

Comment:
This class already has no implementations, so maybe it can be converted to an interface.

Reply:
Client will new object of BaseMessageHandler